index.d.ts

```diff
@@ -104,7 +103,7 @@ export class NodeVM extends EventEmitter {
    * @param {string} [filename] File name (used in stack traces only).
    * @param {Object} [options] VM options.
    */
-  static file(filename: string, options: NodeVMOptions): NodeVM
+  static file(filename: string, options: NodeVMOptions): NodeVM;
```

Comment:
Return type is actually wrong, same for code, both are `any` since they run the code and return what run returned.

Reply:
Ok, will fix. The JSDoc for those functions are wrong...